GitAutomation/GitAutomation.psd1

```diff
@@ -16,7 +16,7 @@
     RootModule = 'GitAutomation.psm1'
 
     # Version number of this module.
-    ModuleVersion = '0.12.0'
+    ModuleVersion = '0.12.3'

```

Comment:
Your changes add some new functionality (new `Credential` parameter for `Receive-GitCommit`) so the version should be bumped to `0.13.0`.

Reply:
This is because I temporarily publish to a personal PS Gallery - once this fork is no longer needed, I don't want to get into trouble with version conflicts/duplicates when heading back to the public one, so please expect another intermediate version before the final commit (of cource including the requested change to `0.13.0`) for this PR ;)

Reply:
While updating the release notes in [`GitAutomation.psd1`](https://github.com/webmd-health-services/GitAutomation/pull/33/files/5dad860ce65ea8b059c1b22e6e5f9698c6b1f822#diff-74fea29414e648f6810cec108f372400), I came across [RELEASE_NOTES.md](https://github.com/webmd-health-services/GitAutomation/blob/develop/RELEASE_NOTES.md) which seems to be quite outdated or even orphaned. Do you want me to catch up/update that one as well or are you about to remove it anyway?

Reply:
Our new pattern is to only document the release notes in the module manifest file since the [PowerShell Gallery](https://www.powershellgallery.com/packages/GitAutomation/0.12.0) automatically pulls the release notes from that location and we also pull those notes when creating the GitHub release. We will be removing the [RELEASE_NOTES.md](https://github.com/webmd-health-services/GitAutomation/blob/develop/RELEASE_NOTES.md) file next time we work in this module.

Reply:
Ok I'll leave [RELEASE_NOTES.md](https://github.com/webmd-health-services/GitAutomation/blob/develop/RELEASE_NOTES.md) unchanged.